pkg/collector/corechecks/system/file_handles_freebsd.go

```diff
@@ -0,0 +1,67 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2019 Datadog, Inc.
+// +build freebsd
+
+package system
+
+import (
+	"github.com/DataDog/datadog-agent/pkg/aggregator"
+	"github.com/DataDog/datadog-agent/pkg/autodiscovery/integration"
+	"github.com/DataDog/datadog-agent/pkg/collector/check"
+	core "github.com/DataDog/datadog-agent/pkg/collector/corechecks"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+	"github.com/blabber/go-freebsd-sysctl/sysctl"
```

Comment:
this repository doesn't seem to have a valid license. Is there an alternative available? (or in the std lib)

Reply:
I doubt there is any alternative. I asked author if there is a way to add a licence.

Reply:
@olivielpeau author added a LICENCE but the `invoke generate-licenses` does not change anything. I'm guessing because LICENCE is not a standard one. Can you help how to include it anyways?

Reply:
Hi @urosgruber, unfortunately we can't use dependencies with licenses that are not OSI-approved (see the complete list of OSI-approved licenses [here](https://opensource.org/licenses/alphabetical)). Datadog open source products can only include dependencies with Apache 2 compatible licenses that are in the OSI list linked above.

You can either ask the dependency author to change to a similarly permissive license that is OSI-approved, find an alternative dependency or avoid using the dependency altogether if possible.

Reply:
@mx-psi author agreed to update the licence. I'll update this PR with the latest commit and hopefully generate-licence will update Licence file as well.

Reply:
Awesome, glad you could sort this out!